src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/CollectionExerciseService.java

```diff
@@ -419,172 +413,6 @@ private CollectionExercise newCollectionExerciseFromDTO(CollectionExerciseDTO co
-   * @param survey SurveyDTO representing survey of collection exercise
-   */
-  private void createActionPlans(CollectionExercise collectionExercise, SurveyDTO survey) {
-    log.with("collection_exercise_id", collectionExercise.getId())
-        .with("survey_id", survey.getId())
-        .debug("Creating action plans for exercise");
-
-    switch (survey.getSurveyType()) {
-      case Business:
-        createDefaultActionPlan(survey, "B");
-        createDefaultActionPlan(survey, "BI");
-        createOverrideActionPlan(collectionExercise, survey, "B");
-        createOverrideActionPlan(collectionExercise, survey, "BI");
-        break;
-
-      case Social:
-        createDefaultActionPlan(survey, "H");
-        createOverrideActionPlan(collectionExercise, survey, "H");
-        break;
-
-      case Census:
-      default:
-        throw new RuntimeException("Census surveys not supported... yet!");
-    }
-
-    log.with("collection_exercise_id", collectionExercise.getId())
-        .with("survey_id", survey.getId())
-        .debug("Successfully created action plans for exercise");
-  }
-
-  /**
-   * Create default action plan for collection exercise and case type if not already exists
-   *
-   * @param survey DTO Representation of survey
-   * @param sampleUnitType Sample Unit Type i.e. (B, H, HI)
-   */
-  private void createDefaultActionPlan(SurveyDTO survey, String sampleUnitType) {
-    log.with("sample_unit_type", sampleUnitType)
-        .with("survey_id", survey.getId())
-        .debug("Creating default action plan");
-
-    // If a casetypedefault already exists for this survey/sampleUnitType do nothing
-    CaseTypeDefault existingCaseTypeDefault =
-        caseTypeDefaultRepo.findTopBySurveyIdAndSampleUnitTypeFK(
-            UUID.fromString(survey.getId()), sampleUnitType);
-    if (existingCaseTypeDefault != null) {
-      log.with("sample_unit_type", sampleUnitType)
-          .with("survey_id", survey.getId())
-          .debug("Default action plan already exists");
-      return;
-    }
-
-    // Create new action plan and associated casetypedefault
-    String shortName = survey.getShortName();
-    String name = String.format("%s %s", shortName, sampleUnitType);
-    String description = String.format("%s %s Case", shortName, sampleUnitType);
-    ActionPlanDTO actionPlan = actionSvcClient.createActionPlan(name, description, null);
-    createCaseTypeDefault(survey, sampleUnitType, actionPlan);
-    log.with("sample_unit_type", sampleUnitType)
-        .with("survey_id", survey.getId())
-        .with("action_plan_id", actionPlan.getId())
-        .debug("Successfully created default action plan");
-  }
-
-  /**
-   * Create case type default for action plan, survey and sample unit type if not already exists
-   *
-   * @param survey DTO Representation of survey
-   * @param sampleUnitType Sample Unit Type
-   * @param actionPlan DTO Representation of action plan
-   */
-  private void createCaseTypeDefault(
```

Comment:
Again not sure but don't we want to create case type based on active enrolment if survey and sample unit type not already exists?

Reply:
umm, can you show me how it's used in the new world?  I took another look and all the caseType stuff refers to actionPlans and is only used in relation to actionplans (which don't exist anymore)